public/js/app/pages/CourseMemo.jsx

```diff
@@ -188,7 +188,7 @@ function CourseMemo() {
         url={sourceUrl}
         startDate={memo.firstTuititionDate}
       />
-      <Row className="p-print-4">
+      <Row className={`p-print-4 ${isMemoArchived()}` ? 'archivedMemoPage' : ''}>
```

Comment:
Is this row really doing what we want?
I was expecting something like this.
```suggestion
      <Row className={`p-print-4 ${isMemoArchived() ? 'archivedMemoPage' : ''}`>
```